Add small Label option (#946)
primer/brand: .changeset/rich-rules-fix.md

---
'@primer/react-brand': patch
---

> Added a new `small` size variant to `Label`

Reviewer: Nit to avoid using 'variant'. A link to the example might be nice here too.

```suggestion
Added a new `small` size option for `Label`

🔗 [See example](https://primer.style/brand/components/Label/react#sizes)
```
